src/Symfony/Bridge/Twig/composer.json

```diff
@@ -26,7 +26,7 @@
         "symfony/asset": "~3.4|~4.0",
         "symfony/dependency-injection": "~3.4|~4.0",
         "symfony/finder": "~3.4|~4.0",
-        "symfony/form": "^4.3.4",
+        "symfony/form": "^4.3.5",
```

Comment:
can be reverted (same below)

Reply:
4.3.4 doesn't have https://github.com/symfony/symfony/pull/33688 so `row_attr` can not exist. I guess the bump + `|default({})` removal in `form_div_layout` should have been made there actually.